README.md

```diff
@@ -26,11 +26,24 @@ npm i fastify-raw-body@2.0.0
 This plugin will add the `request.rawBody`.  
 It will get the data using the [`preParsing`](https://github.com/fastify/fastify/blob/master/docs/Hooks.md#preparsing) hook.
 
+### As a CommonJS module
+
 ```js
 const Fastify = require('fastify')
-const app = Fastify()
+const fastifyRawBody = require('fastify-raw-body')
+```
+
+### As a TypeScript / ES6 module
+
+```js
+import { FastifyInstance as Fastify } from "fastify";
```

Comment:
This throws an error to me:

```
import { FastifyInstance as Fastify } from 'fastify'
         ^^^^^^^^^^^^^^^
SyntaxError: The requested module 'fastify' does not provide an export named 'FastifyInstance'
```

Meanwhile `import Fastify from 'fastify'` works.

(I used `node --experimental-modules try.mjs` with node 12.x)

What was your setup?